indico/modules/events/templates/display/indico/meeting.html

```diff
@@ -5,9 +5,13 @@
 
 {{ render_event_header_msg(event) }}
 
+{% set has_subheader = event.description or files or folders or lectures or conf.getOrgText()
```

Comment:
But this is awful. Can't we change the structure of the HTML instead? I think it would be a pity to have that condition there.